Students/Pintea Beniamin - Samuel/LabWork4/Inventory/src/UTCN/HW/Order.java

```diff
@@ -0,0 +1,39 @@
+package UTCN.HW;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Order {
+
+    String ID;
+    List<Product> orderBasket = new ArrayList<Product>();
+
+    public void setID(String ID) {
+        this.ID = ID;
+    }
+
+    public void addToCart(Product product, int quantity, Inventory stock) {
```

Comment:
stock-ul puteati sa-l modifici din exteriorul metodei `addToCart`. eu as fi facut ca metoda asta sa returneze true/false si in exterior unde era apelata puteai sa faci update la stock in functie de raspunsul primit